src/chrome/content/rules/Matt_Wilcox.net.xml

```diff
@@ -1,10 +1,15 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://mattwilcox.net/ => https://mattwilcox.net/: (60, 'SSL certificate problem: self signed certificate in certificate chain')
+Fetch error: http://www.mattwilcox.net/ => https://www.mattwilcox.net/: (60, 'SSL certificate problem: self signed certificate in certificate chain')
+
```

Comment:
Running

```
openssl s_client -connect www.mattwilcox.net:443 -showcerts
```

Shows that the cert chain includes the self-signed StartCom root cert, which is incorrect (that cert is either in the root store or not, including it does nothing), but doesn't generate a warning in browsers. Considering it a fatal error is a bug in libcurl; I'll file it.

I'll revert this rule for now.